editor/action_map_editor.cpp

```diff
@@ -1249,7 +1253,7 @@ ActionMapEditor::ActionMapEditor() {
 	action_tree->set_column_custom_minimum_width(2, 50 * EDSCALE);
 	action_tree->connect("item_edited", callable_mp(this, &ActionMapEditor::_action_edited));
 	action_tree->connect("item_activated", callable_mp(this, &ActionMapEditor::_tree_item_activated));
-	action_tree->connect("button_pressed", callable_mp(this, &ActionMapEditor::_tree_button_pressed));
+	action_tree->connect("button_mouse_clicked", callable_mp(this, &ActionMapEditor::_tree_button_pressed));
```

Comment:
```suggestion
	action_tree->connect("button_clicked", callable_mp(this, &ActionMapEditor::_tree_button_pressed));
```
Seems like you forgot to update the code after renaming this signal. There are lots of instances like this.

Reply:
Fixed and fix a missing button_pressed -> button_clicked rename.